var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -35,36 +34,20 @@
 
 class Jdk(Package):
     """The Java Development Kit (JDK) released by Oracle Corporation
-       in the form of a binary product aimed at Java developers."""
+       in the form of a binary product aimed at Java developers.
+
+      For legal reasons, Spack is not able to auto-download this tarball.
+      To install, do as follows:
+        1. Set up a Spack mirror.
+        2. Manually download the JDK tarball and put it in the Spack mirror.
```

Comment:
This should mention `var/spack/cache` as the default location; those using mirrors probably know that they're doing so.